src/rest/connector/acisdk_implementation.py

```diff
@@ -0,0 +1,285 @@
+from functools import wraps
+from importlib import import_module
+from logging import getLogger
+from pprint import pformat
+from re import search
+import urllib3
+
+from pyats.connections import BaseConnection
+from rest.connector.utils import get_username_password, verify_apic_version
+
+# create a logger for this module
+log = getLogger(__name__)
+
+
+class AciCobra(BaseConnection):
+    """ACI SDK (Cobra) Implementation for APIC
+
+    Implementation of ACI SDK connection to devices
+    based on pyATS BaseConnection for APIC
+
+    YAML Example
+    ------------
+
+        devices:
+            apic1:
+                connections:
+                    cobra:
+                        class: rest.connector.Acisdk
```

Comment:
should be below, right? please change where applicable.
```suggestion
                        class: rest.connector.AciCobra
```

Reply:
In __init__.py class Acisdk is instantiated (the class imported by rest.connector.Acisdk syntax), which simply redirects to rest.connector.acisdk_implementation.AciCobra by way of `super().__init__`
I used this approach for uniformity (using `rest.connector.<name>` or `from rest.connector import <name>`)
So, Acisdk is the correct class to import, AciCobra (different name) is simply used to not have the same name in both files.
Another approach would be to have the same class name for both of them

Reply:
thank for the reply. got it.